From ac4f795f2c0a81d5b55ce825f3d4c933fbc9e44c Mon Sep 17 00:00:00 2001
From: Pierre-Jean Texier <pjtexier@koncepto.io>
Date: Tue, 4 Jun 2019 11:43:47 +0200
Subject: [libubootenv][PATCH] uboot_env: remove <pthread.h> header file

<pthread.h> include doesn't seem to be necessary inside the library.
So, drop the header completely.

Signed-off-by: Pierre-Jean Texier <pjtexier@koncepto.io>
[Upstream status: http://patchwork.ozlabs.org/patch/1109822/]
---
 src/uboot_env.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/uboot_env.c b/src/uboot_env.c
index e8483bf..3f1bb3b 100644
--- a/src/uboot_env.c
+++ b/src/uboot_env.c
@@ -24,7 +24,6 @@
 #include <fcntl.h>
 #include <errno.h>
 #include <ctype.h>
-#include <pthread.h>
 #include <signal.h>
 #include <sys/file.h>
 #include <sys/types.h>
-- 
2.7.4

